packages/components/grid/src/stories/selection.stories.ts

```diff
@@ -19,6 +24,8 @@ export default {
   }
 };
 
+Icon.register(faCopy, faTrash);
+
 export const ClickableRow: Story = {
```

Comment:
When we are able to click and select a row with mouse, I guess we should be able to select a row also with keyboard? Right now it's not possible

![image](https://github.com/user-attachments/assets/addf6149-8b32-4671-9b03-e38fdabb1708)